scripts/mutect2_wdl/unsupported/funcotator.wdl

```diff
@@ -0,0 +1,153 @@
+# Run Funcotator on a set of called variants from Mutect 2.
+#
+# Description of inputs:
+#
+#   Required:
+#     gatk_docker                -  GATK Docker image in which to run
+#     ref_fasta                  -  Reference FASTA file.
+#     ref_fasta_index            -  Reference FASTA file index.
+#     ref_fasta_dict             -  Reference FASTA file sequence dictionary.
+#     variant_vcf_to_funcotate   -  Variant Context File (VCF) containing the variants to annotate.
+#     reference_version          -  Version of the reference being used.  Either `hg19` or `hg38`.
+#     output_vcf_name            -  Path to desired output file (`.out.vcf` will be appended to this value).
+#
+#   Optional:
+#     data_sources_tar_gz        -  Path to tar.gz containing the data sources for Funcotator to create annotations.
+#     transcript_selection_mode  -  Method of detailed transcript selection.  This will select the transcript for detailed annotation (either `CANONICAL` or `BEST_EFFECT`).
+#     transcript_selection_list  -  Set of transcript IDs to use for annotation to override selected transcript.
+#     annotation_defaults        -  Annotations to include in all annotated variants if the annotation is not specified in the data sources (in the format <ANNOTATION>:<VALUE>).  This will add the specified annotation to every annotated variant if it is not already present.
+#     annotation_overrides       -  Override values for annotations (in the format <ANNOTATION>:<VALUE>).  Replaces existing annotations of the given name with given values.
+#     gatk4_jar_override         -  Override Jar file containing GATK 4.0.  Use this when overriding the docker JAR or when using a backend without docker.
+#
+# This WDL needs to decide whether to use the ``gatk_jar`` or ``gatk_jar_override`` for the jar location.  As of cromwell-0.24,
+# this logic *must* go into each task.  Therefore, there is a lot of duplicated code.  This allows users to specify a jar file
+# independent of what is in the docker file.  See the README.md for more info.
+#
+workflow Funcotator {
+    String gatk_docker
+    File ref_fasta
+    File ref_fasta_index
+    File ref_dict
+    File variant_vcf_to_annotate
+    String reference_version
+    String output_vcf_name
+
+    String? data_sources_tar_gz
+    String? transcript_selection_mode
+    Array[String]? transcript_selection_list
```

Comment:
Please file an issue to make this a `File` that is a list of transcripts.

Reply:
Filed as issue #4090